src/leanback/java/com/archos/mediacenter/video/leanback/MainFragment.java

```diff
@@ -413,7 +413,7 @@ private Box buildAllMoviesBox() {
         }
         else {
             // fallback to regular default icon
-            return new Box(Box.ID.ALL_MOVIES, getString(R.string.all_movies), R.drawable.movies_banner);
+            return new Box(Box.ID.ALL_MOVIES, getString(R.string.all_movies), R.drawable.filetype_new_video);
```

Comment:
This change will result when you have no movie/serie to have an ugly icon at start instead a default colorful one.
I would prefer keeping the old behavior.